<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1083</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="videoEditTab">
       <attribute name="title">
        <string>Редактор видео</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QWidget" name="videoEditWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="setsTab">
       <attribute name="title">
        <string>Наборы</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QWidget" name="setsWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="datasetsTab">
       <attribute name="title">
        <string>Датасеты</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QWidget" name="datasetsWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="modelsTab">
       <attribute name="title">
        <string>Модели</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="trainModelTab">
       <attribute name="title">
        <string>Тренировка модели</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="testModelTab">
       <attribute name="title">
        <string>Тест модели</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="labelsBaseTab">
       <attribute name="title">
        <string>Метки</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QWidget" name="labelsBaseWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1083</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Параметры</string>
    </property>
    <addaction name="actionPath"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPath">
   <property name="text">
    <string>Рабочий каталог</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
